website/components/BlogHeader.tsx

```diff
@@ -25,12 +25,13 @@ function Authors({ data }: { data: BlogPostMeta }) {
     return null;
   }
 
-  const author = CLOUDQUERY_TEAM[authorName];
+  const authorNames = authorName.split(",").map((name) => name.trim());
+  const authors = authorNames.map((name) => CLOUDQUERY_TEAM[name]);
 
   return (
     <div className="w-full border-b border-gray-400 authors border-opacity-20">
       <div className="flex flex-wrap justify-center py-8 mx-auto gap-7">
-        <Avatar {...author} />
+        {authors.map((author) => (<Avatar {...author} />))}
```

Comment:
```suggestion
        {authors.map((author) => (<Avatar key={author.name} {...author} />))}
```

I think that you need to specify a key when rendering arrays in React

Reply:
Ah yeah, good catch